spec/unit/app/models/edition/workflow_spec.rb

```diff
@@ -26,6 +26,26 @@
           expect(edition).to be_published
         end
       end
+
+      %i[draft awaiting_2i].each do |state|
+        context "when the in-progress state '#{state}'" do
```

Comment:
v minor: Should this say: context "when **in** the in-progress state '#{state}'" do